### ISSUE TITLE
Inline @lint has strange behavior depending on code after it

### ISSUE THREAD

**Issue by jclem** [rrrene/credo#177]
``` elixir
defmodule Foo do
  @lint false
  @attr "ABCDEFGABCDEFGABCDEFGABCDEFGABCDEFGABCDEFGABCDEFGABCDEFGABCDEFGABCDEFGAB"
end
```

Will still show a warning for line length. However, this won't:

``` elixir
defmodule Foo do
  @lint false
  @attr "ABCDEFGABCDEFGABCDEFGABCDEFGABCDEFGABCDEFGABCDEFGABCDEFGABCDEFGABCDEFGAB"

  alias Bar
end
```

**Comment by rrrene** [comment]
That's strange indeed. Will try to investigate over the weekend. Thx for reporting! :+1:

**Comment by jclem** [comment]
Thanks, you're welcome!

**Comment by sstern6** [comment]
Confirmed in latest credo, looking into this!

**Comment by sstern6** [comment]
@rrrene so ive been looking into this and for both examples provided above, the correct information gets sent to `issue_for` in `max_line_length.ex`, but for some reason when the alias has a name and is below the error isnt outputted, still looking into that.

Another problem IMO:   
```
defmodule Foo do
  def some_fun(arg0, arg1, arg2, arg3, arg4) do
    "ABCDEFGABCDEFGABCDEFGABCDEFGABCDEFGABCDEFGABCDEFGABCDEFGABCDEFGABCDEFGAB"
  end
end
```

When a string is not assigned to a variable, the error is not outputted.

Let me know your thoughts

**Comment by rrrene** [comment]
I am closing this, since the bug/behaviour/functionality described here is related to `@lint` attributes, which will be removed in favor of more flexible comment-based syntax.

Please refer to #291 for the related discussion :+1:
